packages/server/graphql/public/mutations/loginWithGoogle.ts

```diff
@@ -28,8 +29,14 @@ const loginWithGoogle: MutationResolvers['loginWithGoogle'] = async (
     return {error: {message: 'Email is too long'}}
   }
 
-  const existingUser = await getUserByEmail(email)
+  const [existingUser, samlURL] = await Promise.all([
+    getUserByEmail(email),
+    getSAMLURLFromEmail(email, false)
+  ])
 
+  if (samlURL) {
+    return {error: {message: 'Please login using SSO'}}
```

Comment:
Do you think we should be more descriptive and tell them, something like "Your organization administrator requires you to login with SSO. Please click the "Sign in with SSO" but in the form below."

Reply:
other options would be to change the screen to SSO and enter email or pop SSO window to the url in the SAML table entry for that account along with the message "Your organization administrator requires you to login with SSO. Please click the "Sign in with SSO" but in the form below."

Reply:
I also would also opt for redirecting to SSO sign in with message "Your organization requires you to login with SSO"

Reply:
From Squad 3 meeting, a recommendation "Your organization requires you to login with SSO. Reach out to support@parabol.co for more help!"

If we start getting lots of emails, then that might be signal to better automate this process of mapping a user to the approval POC.